example/analysis_options.yaml

```diff
@@ -0,0 +1,28 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```

Comment:
You can delete the automatically generated comments code as they are unnecessary.
And if it is the same lint rule as analysis_options.yaml located in the root of this project, you can delete this file.